src/WebAPI/appsettings.json

```diff
@@ -31,5 +31,8 @@
     "ClientId": "<Add to secrets>",
     "ClientSecret": "<Add to secrets>",
     "TenantId": "<Add to secrets>"
+  },
+  "AzureMaps": {
+    "Key": "<Add to secrets>"
```

Comment:
Can we update our `settings.json` in Keeper so that everyone can get this functionality locally?

Reply:
This is a good catch, as it reminded me we need the infra for this as well. I've added #762 to cover this which will also include updating Keeper.

I think the tasks in there should be addressed in this PR. However:

1. That would make an already relatively large/complex PR even bigger
2. We will not get that done in time for DDD

As per our conversation on Teams, we will deploy this as is now and I will ClickOps my key into Key Vault for now. We can address the remaining tasks in #762 in a future PR.